blazor/mention/customization.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Customization in Blazor Mention Component | Syncfusion
+description: Checkout and learn here all about customization in Syncfusion Blazor Mention component and much more.
+platform: Blazor
+control: Mention
+documentation: ug
+---
+
+# Customization in Blazor Mention Component
+
+## Show or hide mention character
+
+You can show the mention character as the prefix of the selected item in the Mention component by using the `ShowMentionChar` property. By default, the `ShowMentionChar` property is set to `false`, which means that the mention character will not be displayed as part of the selected mention item.
```

Comment:
To show the mentioned character along with the text when displaying the selected mention item in the target element, you can set the [ShowMentionChar](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfMention-1.html#Syncfusion_Blazor_DropDowns_SfMention_1_ShowMentionChar) property of the Mention component to `true`. This can be useful in cases where you want to clearly differentiate between the selected mention item and the rest of the text in the Mention component.

Reply:
Addressed